selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/intelligent/CommandsEditor.java

```diff
@@ -76,22 +76,36 @@ public CommandsEditor(
     PageFactory.initElements(seleniumWebDriver, this);
   }
 
-  public static final class CommandsEditorType {
-    public static final String COMMAND_LINE_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[1]";
-    public static final String PREVIEW_URL_EDITOR =
-        "(" + Locators.ORION_ACTIVE_EDITOR_CONTAINER_XPATH + ")[2]";
+  public enum CommandsEditorType {
```

Comment:
IMHO this enum describes locators, so the name `CommandsEditorLocator` is more precise, as for me.